src/logger/MeetingSessionPOSTLogger.ts

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import Logger from '../logger/Logger';
+import LogLevel from '../logger/LogLevel';
+import IntervalScheduler from '../scheduler/IntervalScheduler';
+import MeetingSessionConfiguration from '../meetingsession/MeetingSessionConfiguration';
+
+export default class MeetingSessionPOSTLogger implements Logger {
+  private logCapture: string[] = [];
+  private logSequenceNumber: number = 0;
+  private lock = false;
+  private intervalScheduler: IntervalScheduler;
+
+  constructor(
+    private name: string,
+    private configuration: MeetingSessionConfiguration,
+    private batchSize: number,
+    private intervalMs: number,
+    private baseURL: string,
```

Comment:
This should just be `url` as it is the actual URL that is being used in the fetch/sendBeacon requests.